src/android/CodePush.java

```diff
@@ -461,6 +487,8 @@ public void onStart() {
                 handleAppStart();
                 this.markUpdate();
                 this.codePushPackageManager.clearPendingInstall();
+            } else if (codePushReportingManager.hasFailedReport()) {
```

Comment:
but doesn't `notifyApplicationReady` get called by a resume update as soon as it starts as well?